iree/turbine/kernel/wave/runtime/runtime.cpp

```diff
@@ -129,3 +130,35 @@ NB_MODULE(wave_runtime, m)
     m.def("launch", &launch);
     m.def("load_binary", &load_binary);
 }
+
+#else
```

Comment:
Can we just skip this file or the build all together in cmake side if `WAVE_RUNTIME_DISABLED=TRUE`?